syncano/models/incentives.py

```diff
@@ -267,8 +267,7 @@ def run(self, cache_key=None, **payload):
         if self.is_new():
             raise SyncanoValidationError('Method allowed only on existing model.')
 
-        properties = self.get_endpoint_data()
-        endpoint = self._meta.resolve_endpoint('run', properties)
```

Comment:
In:
```
    def resolve_endpoint(self, name, properties):
        endpoint = self.get_endpoint(name)

        for name in endpoint['properties']:
            if name not in properties:
                raise SyncanoValueError('Request property "{0}" is required.'.format(name))

        return endpoint['path'].format(**properties)
```

is `get_endpoint` which is:
```
    def get_endpoint(self, name):
        if name not in self.endpoints:
            raise SyncanoValueError('Invalid path name: "{0}".'.format(name))
        return self.endpoints[name]
```
Maybe we should add check in `resolve_endpoint` for that? Would be much cleaner. What do you think?